libs/output-mapping/phpunit.xml.dist

```diff
@@ -49,7 +49,6 @@
         </testsuite>
         <testsuite name="new-native-types">
             <file>tests/Writer/TableDefinitionV2Test.php</file>
-            <file>tests/TableLoaderTest.php</file>
```

Comment:
Neni potreba, protoze to co to testovalo jsem presunul do TableCreator a ma to vlastni test